src/Sdk/Cerbos/Sdk/CheckResult.cs

```diff
@@ -17,15 +18,28 @@ namespace Cerbos.Sdk
     public class CheckResult
     {
         private readonly Dictionary<string, Effect> _effects;
+        public CheckResourcesResponse.Types.ResultEntry.Types.Resource Resource { get; }
+        public CheckResourcesResponse.Types.ResultEntry.Types.Meta Meta { get; }
+        public List<OutputEntry> Outputs { get; }
 
-        public CheckResult(MapField<string, Effect> effects)
+        public CheckResult(CheckResourcesResponse.Types.ResultEntry result)
```

Comment:
Does this constructor need to be `public`? Aren't the instances of this class only created by the internal SDK code?